helsingborg-stad/helsingborg-io-sls-api
# docs: new github multi theme readme logo

## Explain the changes you’ve made
Change logo in README.md to support both dark and light theme in github.

## Explain why these changes are made
Not supporting github dark theme these days is a big shame for Helsingborgs citizens!  

## Explain your solution
Copy/Paste from our developer test repo.

## How to test the changes?
- Click your profile pic in the upper right corner in github.
- Select "Settings"
- Select "Appearance"
- Switch themes.
- [Click here to check the result](https://github.com/helsingborg-stad/helsingborg-io-sls-api/tree/docs/update-readme-logo)

## Anyhting else? (optional)
Nope!
